### DISCUSSION TITLE
Hi everyone , need some healp receveing location

### DISCUSSION THREAD

**Discussion by victormedranop** [go-telegram-bot-api/telegram-bot-api#647] [Q&A]
if someone can share a sample code will be apreciated.

thanks

Victor

**Comment by Njunwa1** [comment]
Just visit the helpers.go, there you will find the Keyboardbutton able to request user location

https://github.com/go-telegram-bot-api/telegram-bot-api/blob/4126fa611266940425a9dfd37e0c92ba47881718/helpers.go#L647C1-L653C2

```
// NewKeyboardButtonLocation creates a keyboard button that requests
// user location information upon click.
func NewKeyboardButtonLocation(text string) KeyboardButton {
	return KeyboardButton{
		Text:            text,
		RequestLocation: true,
	}
}
```
